### ISSUE TITLE
lm-evaluation-harness dep bump breaks build

### ISSUE THREAD

**Issue by kremlin-** [kingoflolz/mesh-transformer-jax#144]
hi,

lm-evaluation-harness specifies tensorflow-estimator version incompatible with this repo's specified version of tf-cpu, causes dep error:

```
$ pip3 install -r requirements.txt
...
The conflict is caused by:
    lm-eval 0.0.1 depends on tensorflow-estimator==2.6.0
    tensorflow-cpu 2.5.2 depends on tensorflow-estimator<2.6.0 and >=2.5.0
    lm-eval 0.0.1 depends on tensorflow-estimator==2.6.0
    tensorflow-cpu 2.5.1 depends on tensorflow-estimator<2.6.0 and >=2.5.0
    lm-eval 0.0.1 depends on tensorflow-estimator==2.6.0
    tensorflow-cpu 2.5.0 depends on tensorflow-estimator<2.6.0 and >=2.5.0rc0
```

**Comment by kremlin-** [comment]
```diff
diff --git a/requirements.txt b/requirements.txt
index 4851e8e..092278e 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -6,7 +6,7 @@ requests~=2.25.1
 fabric~=2.6.0
 optax==0.0.6
 git+https://github.com/deepmind/dm-haiku
-git+https://github.com/EleutherAI/lm-evaluation-harness/
+git+https://github.com/EleutherAI/lm-evaluation-harness@c406a62047
 ray[default]==1.4.1
 jax~=0.2.12
 Flask~=1.1.2
```

quick fix

**Comment by JialuZhang** [comment]
@kremlin- 

I got the same error when I tried to run the experiment on google colab.

Here is the error detail.
###
ERROR: Cannot install -r mesh-transformer-jax/requirements.txt (line 14) and -r mesh-transformer-jax/requirements.txt (line 9) because these package versions have conflicting dependencies.

The conflict is caused by:
lm-eval 0.0.1 depends on tensorflow-estimator==2.6.0
tensorflow-cpu 2.5.2 depends on tensorflow-estimator<2.6.0 and >=2.5.0
lm-eval 0.0.1 depends on tensorflow-estimator==2.6.0
tensorflow-cpu 2.5.1 depends on tensorflow-estimator<2.6.0 and >=2.5.0
lm-eval 0.0.1 depends on tensorflow-estimator==2.6.0
tensorflow-cpu 2.5.0 depends on tensorflow-estimator<2.6.0 and >=2.5.0rc0

To fix this you could try to:

loosen the range of package versions you've specified
remove package versions to allow pip attempt to solve the dependency conflict
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
Processing ./mesh-transformer-jax
###

Do you know how to run the GPT-J model on colab? Many thanks.

**Comment by kremlin-** [comment]
@JialuZhang 

you could clone this repo locally, edit line 9 of requirements.txt to:

git+https://github.com/EleutherAI/lm-evaluation-harness@c406a62047

 upload it to your gdrive, mount gdrive in colab, make new code window, enter:
!pip3 install -r <path_to_gdrive_mount>/mesh-transformer-jax/requirements.txt

and run that code window

**Comment by kingoflolz** [comment]
Fixed by 27a8036b9f8d7d68332a7bc8794f968d5bbd1dbb
